docs/oathkeeper/index.md

```diff
@@ -100,10 +103,10 @@ request to ORY Oathkeeper.
 
 ### Access Control Decision API
 
-The Access Control Decision API is exposed at `/decision` of the ORY Oathkeeper
-API port. It follows best-practices and works with most (if not all) modern API
-Gateways and Reverse Proxies. The endpoint matches every sub-path and HTTP
-Method:
+The ORY Oathkeeper Access Control Decision API follows best-practices and works with most (if not all) 
+modern API gateways and reverse proxies.
+To verify a request, send it to the `decision` endpoint.
+It matches every sub-path and HTTP Method:
 
 - `GET /decision/v1/api`
```

Comment:
I actually noticed that the endpoint is `/decisions`, not `/decision`. While you're at it would you mind updating this?

Reply:
done